tests/karate/src/test/java/be/election/electionSetup.feature

```diff
@@ -3,21 +3,22 @@ Feature: Setup an Election
   Background:
         # This is feature will be called to test Election Setup
         # For every test a file containing the json representation of the message is read
-        # and is sent to the backend this is done via :
+        # then sent to the backend and stored there.
+        # This is done via :
         # eval frontend.send(<message>) where a mock frontend sends a message to backend
         # Then the response sent by the backend and stored in a buffer :
         # json response = frontend_buffer.takeTimeout(timeout)
         # and checked if it contains the desired fields with :
         # match response contains deep <desired fields>
 
-    # The following calls makes this feature, mockFrontEnd.feature and server.feature share the same scope
+    # The following calls make this feature, mockFrontEnd.feature and server.feature share the same scope
     * call read('classpath:be/utils/server.feature')
     * call read('classpath:be/mockFrontEnd.feature')
     * def electionSetupId = 4
     * string laoChannel = "/root/p_EYbHyMv6sopI5QhEXBf40MO_eNoq7V_LygBd4c9RA="
 
-  # Testing if after setting up a valid lao, subscribing to it, sending a catchup
-  # creating a valid election setup should result in a valid response from the backend
+  # Testing if after a successful roll call, sending a valid election
+  # setup should result in a valid response from the backend
```

Comment:
I'm really sorry to nitpick but in general "Testing if (...), (...) should return (...)" does not make grammatical sense. Just removing the "should" would be an improvement